### ISSUE TITLE
Thinks what you doing. Did you have plan to support mium,maan,etc series?

### ISSUE THREAD

**Issue by orangeff** [DoctorD1501/JAVMovieScraper#297]

**Comment by ObviouslyNotRealName** [comment]
+1 for more series support.  Could you also provide their official website/sources? So that someone who is capable and wish for the same thing might want to start on it. I'm really really trying to learn Java currently, starting from zero knowledge, just to someday be able to contribute to this project myself.

Other series I would love to be able to scrape are such as "ARA" "LUXU" (Official site is probably here: https://www.mgstage.com) and even "FC2."

**Comment by NewMember12345** [comment]
I agree, I've been manually pulling and inputing these data and it takes me hours to get 20 done.

**Comment by fannysilence** [comment]
you can use this site to get these data
https://www.jav321.com/
